pybluedot/cli.py

```diff
@@ -0,0 +1,7 @@
+"""
+cli.py constructs a cli interface of commands, options, and subcommands in argparse, given data structures from commands.py
+"""
+import argparse
+import json
+from commands import cmd, subcmd 
```

Comment:
Let's use absolute imports starting with `pybluedot` for everything (so this would be `from pybluedot.commands import cmd, subcmd).